src/components/results/result.js

```diff
@@ -13,14 +13,21 @@ class Result extends React.Component {
 
   render() {
     const {bootstrapCss, doc, fields} = this.props;
+    const Withlink = ({field, link, children})=> (
+        field.field === "id"?
+        <a href={link}>{children}</a> :
+        children
+    );
 
```

Comment:
wrapper function to conditionally render fields as links. For right now, I just hardcoded that only the "id" field would be converted to a link.